pkg/agent/agent.go

```diff
@@ -111,10 +128,73 @@ func (a *Agent) Run(healthCheck *metrics.HealthCheck, enableProfiling bool, bind
 	}()
 
 	<-a.ctx.Done()
+
+	_ = a.DeleteNodeResourceTsp()
 }
 
 func getAgentName(nodeName string) string {
-	return nodeName + "_" + string(uuid.NewUUID())
+	return nodeName + "." + string(uuid.NewUUID())
+}
+
+func (a *Agent) CreateNodeResourceTsp() string {
```

Comment:
If the agent restart on a large scale,  it will make an impact to the predict module ？

Reply:
Maybe, but in the case of a lot of nodes, even if you create tsp for each node manually or using ClusterNodePrediction, it will put a lot of pressure on the prediction module.